help/destinations/catalog/advertising/acxiom-audience-distribution.md

```diff
@@ -0,0 +1,143 @@
+
+
+While the segmentation runs, you must set up the connection to a specific destination (e.g., [!DNL LG Ads], [!DNL Spectrum], [!DNL Altrice], etc.).
+
+Once the connection is set up and the audience is created, you can schedule the data flow within the destination card.
+
+## Prerequisites {#prerequisites}
+- **Confirm Terms of Use:** Before you can configure a new [!DNL Acxiom Audience Distribution] destination, you must read and sign [!DNL Acxiom’s] Terms of Use Agreement. You will receive the link to the agreement once your executed sales order is complete. Until you sign the agreement, you will not see the [!DNL Acxiom Audience Distribution] destination card in the Experience Platform destination catalog. After you accept and sign the agreement, [!DNL Adobe] will complete your onboarding process and you will see the [!DNL Acxiom Audience Distribution] destination card.
+- **Know your Adobe organization ID:** Your [!DNL Adobe] organization ID is needed to complete your Terms of User Agreement. See [!DNL Adobe's] *Organizations in Experience Cloud* topic for details on how to [view your organization ID](https://experienceleague.adobe.com/en/docs/core-services/interface/administration/organizations#concept_EA8AEE5B02CF46ACBDAD6A8508646255).
+
+
+## Supported Destinations {#supported-destinations}
+
+[!DNL Acxiom’s Audience Distribution] destination currently supports audience activation to the following platforms.<br> 
+	
+- [!DNL Altice]
+- [!DNL Ampersand]
+- [!DNL Comcast]
+- [!DNL Cox]
+- [[!DNL LG Ads]](#lg-ads)
+- [!DNL Spectrum]
+- [!DNL Viant]
+
+
+## Connect to a new destination {#connect}
+Authentication to [!DNL Acxiom's Audience Distribution] destination is automatically handled behind the scenes for your convenience.
+
+
+## Destinations specific settings {#destination-specific-settings}
+Some [!DNL Acxiom Audience Distribution] destinations require additional information. The sections below provide detailed guidance on how to setup the destinations that have specific configuration options.
+
+### [!DNL LG Ads] {#lg-ads}
+To configure details for the destination, fill in the fields below.
+- **Segment Category**: The target category or vertical that your segment falls into. Example: financial services, automotive, health, etc.
+	
+![LG Ads destination details](../../assets/catalog/data-partner/acxiom/lg_ads_destination_details.png)
+
+
+## Activate audiences to this destination {#activate} 
+>[!IMPORTANT]
+>
+>- To activate data, you need the [!UICONTROL View Destinations], [!UICONTROL Activate Destinations], [!UICONTROL View Profiles], and [!UICONTROL View Segments] [access control permissions](../../../access-control/home.md#permissions). Read the [access control overview](../../../access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+

```

Comment:
Please keep the information from the [template](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template#activate) corresponding to your destination type.

![Screenshot 2024-09-25 at 13 30 39](https://github.com/user-attachments/assets/0d707daf-313c-4523-81f1-4be0254308c8)

Reply:
I replaced this note with your suggested note below.

Reply:
Based on your next comment, I removed this note completely.

Reply:
@lrugge-acxiom Sorry if I wasn't clear enough. This isn't what I meant.

- The IMPORTANT note in the template is MANDATORY. Do not remove that one.
- The second row I higlighted in the image is MANDATORY, because you're building a file-based destination. Please leave that there.

Reply:
Thank you for clarifying. This is done.